source/server/server.cc

```diff
@@ -134,10 +135,10 @@ void InstanceImpl::flushStats() {
   stat_flush_timer_->enableTimer(config_->statsFlushInterval());
 }
 
-int InstanceImpl::getListenSocketFd(uint32_t port) {
+int InstanceImpl::getListenSocketFd(const std::string& address) {
+  Network::Address::InstancePtr addr = Network::Utility::resolveUrl(address);
```

Comment:
I have to, address is in format 'tcp://address:port' while localAddress()->asString() is in format 'address:port'